packages/docusaurus-plugin-content-docs/src/options.ts

```diff
@@ -54,6 +54,8 @@ export const DEFAULT_OPTIONS: Omit<PluginOptions, 'id' | 'sidebarPath'> = {
   sidebarCollapsible: true,
   sidebarCollapsed: true,
   breadcrumbs: true,
+  onBrokenTags: 'warn',
+  tagsFilePath: '',
```

Comment:
`''` should be forbidden by validation

Values:
- `undefined` (default): search `tags.yml` by convention. Feature disabled if file does not exist
- `null/false`: feature disabled even if `tags.yml` exists
- `tagFile.yml`: search for `tagFile.yml` and throw if file does not exist